plutus-playground/plutus-playground-server/app/PSGenerator.hs

```diff
@@ -176,3 +212,9 @@ generate outputDir = do
         myBridgeProxy
         (Proxy @API.API)
     writePSTypes outputDir (buildBridge myBridge) myTypes
+    writeUsecases outputDir
+
+k :: IO ()
+k =
+    generate
+        "/Users/kris/Work/Clients/IOHK/plutus/plutus-playground/plutus-playground-client/src"
```

Comment:
That looks like it will only work on one machine, and also what is `k` ? `kris`?

Reply:
deleted